packages/@lwc/rollup-plugin/src/__tests__/fixtures/expected_compat_config_simple_app.js

```diff
@@ -144,7 +144,7 @@
       key: 0
     }, [api_custom_element("x-foo", _xFoo, {
       props: {
-        "x": "1"
+        x: "1"
```

Comment:
these are result of this [commit](https://github.com/salesforce/lwc/pull/2256/commits/d4b6c00414fc9b832c5326d364e7ebcea1ff3994#diff-12aaa042aba41d07e9e4fb8e237f284ced0bd1c40cdaf7b611454b9f818d258cL450-R458) (instead of `t.literal` via `objectToAST`, now is `t.identifier`).

Reply:
In case if the prop name is a valid JavaScript identifier, I think it is safer to move it back to a literal. 
Terser will turn the literal back in an identifier if the literal is a valid JavaScript identifier.

Reply:
I don't understand this; could someone explain?